Common/Orders/OrderTypes.cs

```diff
@@ -56,11 +56,10 @@ public enum OrderType
         OptionExercise
     }
 
-
     /// <summary>
-    /// Time In Force - defines the length of time over which an order will continue working before it is canceled
+    /// Time In Force Type - defines the length of time over which an order will continue working before it is canceled
     /// </summary>
-    public enum TimeInForce
+    public enum TimeInForceType
```

Comment:
Is this enum still required? I think the only place it's used anymore is type checks which can be replaced by actual type checks vs enum value checks. The other place it's currently used is the implicit operator, but I don't think that one is required since users can use `TimeInForce.Day` instead of `TimeInForceType.Day` -- the concern is having two very similar things that are potentially different.

Reply:
👍  Agree, the `TimeInForceType` enum is not really needed any more -- I'll remove it